mypy/messages.py

```diff
@@ -872,6 +872,9 @@ def unimported_type_becomes_any(self, prefix: str, typ: Type, ctx: Context) -> N
         self.fail("{} becomes {} due to an unfollowed import".format(prefix, self.format(typ)),
                   ctx)
 
+    def explicit_any(self, location: str, ctx: Context) -> None:
+        self.fail('Explicit "Any" is not allowed in a {}'.format(location), ctx)
```

Comment:
I wonder if this error would be clearer if it was just `Explicit "Any" is not allowed.`.  This reads to me like it's implying that the `Any` is not allowed in that particular location, but in fact it's not allowed in _any_ location.